### PR TITLE
Search & paginate users

### PR DESCRIPTION
**Pull request by tcitworld**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | yes
| Fixed tickets | #3050
| License       | MIT

### CODE REVIEW

File: src/Wallabag/UserBundle/Controller/ManageController.php [hunk 1]
```diff
@@ -19,17 +22,32 @@ class ManageController extends Controller
     /**
      * Lists all User entities.
      *
-     * @Route("/", name="user_index")
+     * @Route("/index/{page}", name="user_index")
```
Comment (j0k3r) [inline src/Wallabag/UserBundle/Controller/ManageController.php]: Why not just `"/{page}"`?

Comment (tcitworld) [inline src/Wallabag/UserBundle/Controller/ManageController.php]: Because we have the "/{userid}/edit" route and I thought it would be confusing

Comment (j0k3r) [inline src/Wallabag/UserBundle/Controller/ManageController.php]: Hum I see. I really don't like that `/index/` in url.
Can't you use `/?page=`?

Comment (tcitworld) [inline src/Wallabag/UserBundle/Controller/ManageController.php]: If you want. I don't really like these kind of urls ;)

Comment (j0k3r) [inline src/Wallabag/UserBundle/Controller/ManageController.php]: :bulb: Otherwise, use `/list/` instead of `/index/`

**Comment by j0k3r** [comment]
We might merge both search & list page so the url doesn't change and we only have one method.
What do you think?

Also, I don't think the label is necessary here, since the full description of the field is the placeholder

image

**Comment by tcitworld** [comment]
Suits me.

**Comment by nicosomb** [comment]
* I merged search and list
* I removed `label` and icon for the search form.

File: src/Wallabag/UserBundle/Controller/ManageController.php [hunk 2]
```diff
@@ -169,52 +137,44 @@ private function createDeleteForm(User $user)
...
-
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
             $this->get('logger')->info('searching users');
-            $em = $this->getDoctrine()->getManager();
 
             $searchTerm = (isset($request->get('search_user')['term']) ? $request->get('search_user')['term'] : '');
 
             $qb = $em->getRepository('WallabagUserBundle:User')->getQueryBuilderForSearch($searchTerm);
```
Comment (tcitworld) [inline src/Wallabag/UserBundle/Controller/ManageController.php]: Where is `getQueryBuilderForSearch` ?

Comment (nicosomb) [inline src/Wallabag/UserBundle/Controller/ManageController.php]: In `src/Wallabag/UserBundle/Repository/UserRepository.php`.

Comment (tcitworld) [inline src/Wallabag/UserBundle/Controller/ManageController.php]: I know, I added it myself. But github didn't show it in the diff. o_0

**Pull request merged**